packages/text-input/text-input.scss

```diff
@@ -18,18 +18,15 @@ $bottom-margin: $component-spacing--large;
 $input-total-height: $label-height + $input-height + $bottom-padding;
 $textarea-height--expanded: $input-height * 7 + $bottom-padding;
 
-$error-color: change-color($blomst, $lightness: 45%); // make $blomst WCAG compliant
-
 .jkl-text-field {
     display: block;
     box-sizing: border-box;
     position: relative;
     font-family: inherit;
     width: 100%;
-    height: $input-total-height;
+    min-height: $input-total-height;
     padding-top: $label-height;
     background-color: $input-bg-color;
-    margin-bottom: $component-spacing--large;
```

Comment:
I don't think we should get rid of the margin. Having the spacing as a margin (like now) acts more like a "minimum spacing", as it won't add up with adjacent margins. In the unlikely event that there needs to be less than `1rem` spacing between the input and the element below, it can be overridden by the new spacing API classes.